docs/html-css/week-3/lesson.md

```diff
@@ -20,6 +19,481 @@ Learning objectives for this lesson can be found [here](./learning-objectives.md
+  <option value="polish">Polish</option>
+  <option value="romanian">Romanian</option>
+  <option value="greek">Greek</option>
+  <option value="mexican">Mexican</option>
+  <option value="turkish">Turkish</option>
+  <option value="lebanese">Lebanese</option>
+  <option value="syrian">Syrian</option>
+  <option value="moroccan">Moroccan</option>
+  <option value="ethiopian">Ethiopian</option>
+  <option value="nigerian">Nigerian</option>
+  <option value="indian">Indian</option>
+  <option value="chinese">Chinese</option>
+  <option value="thai">Thai</option>
+  <option value="korean">Korean</option>
+  <option value="japanese">Japanese</option>
+</select>
+:::
+
+:::tip
+Both radio and select inputs allow the user to select one choice out of many.
+When do we know which one to use?
+
+A helpful guideline is if there are only a few options to choose from (less than 5), use radio inputs.
+If there are lots of options to choose from, use a select input.
+:::
+
+### Labels
+
+Every form control **must** have a label.
+Firstly, this is important for users to see what they should enter into the form.
+Secondly, there are visually-impaired users that use screen readers to interact with websites.
+When screen reader users navigate to a form control, the screen reader reads out the label so the user can understand what to enter into the form.
+Without a label, the screen reader user does not get a description of what they are supposed to enter.
+
+There are two ways to associate a `label` with an `input`.
+The first way is by connecting the `label` and `input` using `for` and `id` as we have seen.
+This is the preferred way as it is more explicit.
+
+```html
+<label for="username">Username</label>
+<input id="username" type="text" name="username">
+```
+
+The second way is by wrapping the `input` within the `label`.
+
+```html
+<label>
+  Username
+  <input type="text" name="username">
+</label>
+```
+
+It is also important that every group of radio and checkbox inputs is wrapped in a `fieldset` with a `legend`, which we've seen in the [radio input fields section](#radio-input-fields).
+Again, when screen reader users navigate to this group of inputs, they will understand what the website is asking them to choose.
+
+Another feature of labels is that the user can click on the label to activate the form control.
+This provides a larger clickable area to select the form control, and can be beneficial to users who have motor disabilities.
+
+:::tip
+It is quite common to add placeholders to text inputs.
+Adding only placeholders is bad practise, because it is not guaranteed to be read out by screen readers.
+Default styles for placeholders can also have poor colour contrast, which means visually-impaired users might not be able to read it.
+
+<input style={{width:'100%', marginBottom:'1rem'}} type="text" placeholder="I am a placeholder. Placeholders are not labels so do not use me as a label."/>
+
+In summary, always give every form control a label.
```

Comment:
Perhaps make this bold in case students skim over?...

Reply:
I've put `always` in bold 👍 

At the beginning of this section, we also state this in bold:

> Every form control **must** have a label.